tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+Suite Setup         Set Library Search Order  SeleniumLibrary
+Suite Teardown      End Web Test
+
+
+*** Test Cases ***
```

Comment:
It would be good also to verify if the admin user has the admin privileges and users doesn't and users not allowed can't access

Reply:
I think the test here can be just to check the default configuration and if jupyterhub works fine. There are more complex permission tests in tests/Tests/400__ods_dashboard/410__ods_dashboard_settings.robot and tests/Tests/500__jupyterhub/jupyterhub-user-access.robot